app/controllers/ems_common.rb

```diff
@@ -899,6 +900,12 @@ def retrieve_amqp_security_protocols
     [[_('Non-SSL'), 'non-ssl']]
   end
 
+  def retrieve_container_security_protocols
+    [[_('SSL'), 'ssl-with-validation'],
+     [_('SSL trusting custom CA'), 'ssl-with-validation-custom-ca'],
+     [_('SSL without validation'), 'ssl-without-validation']]
```

Comment:
Yes, but.  Alas, Openstack uses the values
```
[[_('SSL without validation'), 'ssl'], 
 [_('SSL'), 'ssl-with-validation'], 
 [_('Non-SSL'), 'non-ssl']]
```
where just `ssl` means insecure :-(
And AFAICT some other providers use `ssl` to mean secure.
It'd be good to fix, but would require migration(s), I don't want to block on that, and I don't want to add to the confusion now.
That only left the option of using 2 verbose but explicit values.

(If we do migrate openstack later, it'll be safer to never redefine `ssl` but switch to `ssl-{with,without}-validation` too.)